apps/bfd-server/bfd-server-war/src/test/resources/endpoint-responses/eobByPatientIdAll.json

```diff
@@ -2059,14 +2059,14 @@
         "type" : [ {
           "coding" : [ {
             "system" : "https://bluebutton.cms.gov/resources/codesystem/diagnosis-type",
-            "code" : "external-first",
-            "display" : "The code used to identify the 1st external cause of injury, poisoning, or other adverse effect."
+            "code" : "principal",
+            "display" : "The single medical diagnosis that is most relevant to the patient's chief complaint or need for treatment."
           } ]
         }, {
           "coding" : [ {
             "system" : "https://bluebutton.cms.gov/resources/codesystem/diagnosis-type",
-            "code" : "principal",
-            "display" : "The single medical diagnosis that is most relevant to the patient's chief complaint or need for treatment."
+            "code" : "external-first",
+            "display" : "The code used to identify the 1st external cause of injury, poisoning, or other adverse effect."
```

Comment:
I don't think these changes in the output are related to your coding changes and I suspect that the tests will still pass if you revert them. Please try really hard to only change as little as you have to in these files.